spec/requests/api/v2/profiles_controller_update_spec.rb

```diff
@@ -0,0 +1,205 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::V2::ProfilesController do
+  let(:response_json) { JSON.parse(response.body) }
+  let(:access_token) { create(:access_token).token }
+  let(:content_type) { ApiController::CONTENT_TYPE }
+  let(:headers) { { 'CONTENT_TYPE': content_type }.merge('Authorization' => "Bearer #{access_token}") }
+  let(:risk_type_1) { create :assessment_question, :risk }
```

Comment:
Don't usually feel like `<thing>_<n>` is that revealing as far as naming things is concerned. I guess we get value out of creating multiple assessment questions 🤷‍♂️

Reply:
Maybe we could create one and call is `assessment_question` and validate against that?

Reply:
There's nothing different for one or more assessment questions when updating jsonb?

Reply:
not that I'm aware of no, but I think we should add validation for those though, I've discussed this with Jonny, and we'll pick those up in later work next sprint